templates/values.go

```diff
@@ -64,30 +71,43 @@ var DefaultSSHTemplates = SSHTemplates{
 
 // DefaultSSHTemplateData contains the data of the default templates used on ssh.
 var DefaultSSHTemplateData = map[string]string{
-	// include.tpl adds the step ssh config file.
+	// base_config.tpl adds the step ssh config file.
 	//
 	// Note: on windows `Include C:\...` is treated as a relative path.
-	"include.tpl": `Host *
+	"base_config.tpl": `Host *
 {{- if or .User.GOOS "none" | eq "windows" }}
+{{- if .User.Authority }}
+	Include "{{ .User.StepBasePath | replace "\\" "/" | trimPrefix "C:" }}/ssh/includes"
+{{- else }}
 	Include "{{ .User.StepPath | replace "\\" "/" | trimPrefix "C:" }}/ssh/config"
+{{- end }}
+{{- else }}
+{{- if .User.Authority }}
+	Include "{{.User.StepBasePath}}/ssh/includes"
 {{- else }}
 	Include "{{.User.StepPath}}/ssh/config"
+{{- end }}
```

Comment:
This is for backward compatibility with old clients right? Ideally old and new clients should always use the `~/.ssh/includes`. Is this the case?